### ISSUE TITLE
Super user ability - do anything except a few permissions

### ISSUE THREAD

**Issue by thejuan** [stalniy/casl#119]
I've seen there is a wildcard subject 'all', but not for action.
I'd like to create an ability where the user can do anything except a few specified cannot's
Is this currently possible?

```
const ability = AbilityBuilder.define((can, cannot) => {
  cannot('delete', 'Post', { 'comments.0': { $exists: true } })
  can('all','all')
})
```

**Comment by pixiefrog** [comment]
A 'wildcard' for all actions is `manage`. You can do what you asked for, you were already pretty close yourself. Order does matter in this case:

```
const ability = AbilityBuilder.define((can, cannot) => {
  can('manage','all')
  cannot('delete', 'Post', { 'comments.0': { $exists: true } })
})
```

You can read more about it in the [docs](https://stalniy.github.io/casl/abilities/2017/07/20/define-abilities.html) (combining abilities is what you're looking for)

**Comment by stalniy** [comment]
thanks for reply @pixiefrog ! 
Frankly speaking, `manage` is just an alias for CRUD operations (create, read, update, delete). 

@thejuan if your system supports other actions then the current implementation will not help you. What you can do is to extend `Ability` class and implement this functionality. Alternatively you can create a PR :) and I will publish a new version

**Comment by thejuan** [comment]
Awesome, I have an auth wrapper which is doing the work for now.
I'll add a PR to my todo list.

**Comment by stalniy** [comment]
published in `@casl/ability@3.0.0`
